### DISCUSSION TITLE
No module named 'rich', but I am very sure `rich` is installed

### DISCUSSION THREAD

**Discussion by JoshPing** [oobabooga/text-generation-webui#5833] [Q&A]
Mac 12.7, python3, (base) environment
Tried both download and clone the repository from GitHub, after `./start_macos.sh` always come back with "ModuleNotFoundError: No module named 'rich'"
Requirement already satisfied: rich in /opt/miniconda3/lib/python3.12/site-packages (13.7.1)
Requirement already satisfied: markdown-it-py>=2.2.0 in /opt/miniconda3/lib/python3.12/site-packages (from rich) (3.0.0)
Requirement already satisfied: pygments<3.0.0,>=2.13.0 in /opt/miniconda3/lib/python3.12/site-packages (from rich) (2.17.2)
Requirement already satisfied: mdurl~=0.1 in /opt/miniconda3/lib/python3.12/site-packages (from markdown-it-py>=2.2.0->rich) (0.1.2)

Any one can help?

**Comment by OpM3TA** [comment]
same with different project. smh ever figure it out?
